changelog/unreleased/kong/feat-disable-h2-alpn.yml

```diff
@@ -0,0 +1,4 @@
+message: | 
+  **Core**: Added `tls.disable_http2_alpn()` function needed patch for disabling HTTP/2 ALPN when tls handshake.
```

Comment:
```suggestion
  **Core**: Added the API `tls.disable_http2_alpn()` to disable HTTP/2 ALPN dynamically when tls handshake.
```

Reply:
that does not mean we introduce `tls.disable_http2_alpn()` here. We provide the patch in the kong repo which is needed by `lua-kong-nginx-module` PR.